notifier/chatters.rst

```diff
@@ -98,6 +98,44 @@ to add some interactive options called `Block elements`_::
 
     $chatter->send($chatMessage);
 
+Adding Fields and Values to a Slack Message
+-------------------------------------------
+
+To add fields and values to your message you can use the
```

Comment:
I'd reword this intro to explain what "Slack fields" are. E.g.:

```rst
`Slack fields`_ are text-based metadata that can be added to message sections
and are visible to end-users. Use the
:method:`SlackSectionBlock::field() <Symfony\\Component\\Notifier\\Bridge\\Slack\\Block\\SlackSectionBlock::field>`
method to add fields to the message:

.. _`Slack fields`: https://api.slack.com/reference/block-kit/blocks#section_fields
```

Reply:
Done, thanks